### PR TITLE
Create phones.md

### PR DESCRIPTION
**Pull request by sammachin**
Details for 2024

### CODE REVIEW

File: templates/about/phones.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+
+## Getting Started
+
+You can pick a 4-digit extension number to use, and call anyone on site via it. Create your number in the  [Electromagnetic Phonebook](https://phones.emfcamp.org)
+
+Firstly you will need to register an account with a username, email & password. We send an email validation message from [poc@emfcamp.org](mailto:poc@emfcamp.org) so make sure you’ve clicked that before you can login.
+
+### Registering a Number
+
+Once logged in cick Add a Number in the top menu bar.
```
Comment (lukegb) [inline templates/about/phones.md:15]:
```suggestion
Once logged in click Add a Number in the top menu bar.
```

File: templates/about/phones.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+The My Numbers link will show you the numbers under your account,
+
+Next to each number is 3 buttons.
+
+**Details** will show you how to use that number with specific instructions for the Type of Service you are using.
+
+**Modify** lets you amend the Description in the phonebook, change the visibility in the phonebook and set a Fallback Number.
+
+The** Falback Number **is a number that will be called if the main number doesn’t answer the call within 30sec or is unavailable.
```
Comment (lukegb) [inline templates/about/phones.md:44]:
```suggestion
The **Fallback Number** is a number that will be called if the main number doesn’t answer the call within 30sec or is unavailable. 
```

File: templates/about/phones.md [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+
+### DECT 
+
+There's site-wide DECT phone coverage
+
+You can bring your own DECT telephone and join the network, enabling you to call other EMF participants and interactive services for free.
+
+If you're buying a DECT phone and don't know what to get, buy pretty much any Gigaset DECT handset, for example:
+
+	Gigaset A170, £18 on Amazon [https://amzn.to/3yaJ5EG](https://amzn.to/3yaJ5EG)
```
Comment (lukegb) [inline templates/about/phones.md:58]: This is... a Matt Gray affiliate link?

File: templates/about/phones.md [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+
+Gigaset Life A, ~£60 on Amazon [https://amzn.eu/d/4GuVoMu](https://amzn.eu/d/4GuVoMu) (Known to be compatible with Bluetooth Hearing Aids from Phonak, Unitron and Hansaton)
+
+If you've got one already, feel free to bring it. Use the DECT Phone Compatibility List to find out if your phone is likely compatible.
```
Comment (lukegb) [inline templates/about/phones.md:62]: Maybe link to the list?

File: templates/about/phones.md (hunk 3 above)
Comment (sammachin) [reply templates/about/phones.md:58]: LOL I just copied that from the Wiki that Matt put on, do we have an EMF affiliate? if so we should possibly use that but equally I'm fine with Matt getting the 50p as he did the research

**Review by jellybob — APPROVED** [review]

File: templates/about/phones.md (hunk 3 above)
Comment (russss) [reply templates/about/phones.md]: Can we just make this a non-affiliate link for the moment

Comment (MattGrayYes) [reply templates/about/phones.md]: Oh shit I had no idea that the Amazon short urls had my affiliate in them, sorry!
If it’s any consolation I’ve never made any money off it, and don’t even know how to look at the (depersonalised) data!

**Pull request merged**